doc/release-notes.md

```diff
@@ -25,11 +25,17 @@ RPC Changes
   while you should already be checking the async operation status, there are now
   more cases that may trigger failure at that stage.
 - The `AllowRevealedRecipients` privacy policy is now required in order to choose a
-  transparent change address for a transaction. This will only occur when the wallet 
-  is unable to construct the transaction without selecting funds from the transparent 
+  transparent change address for a transaction. This will only occur when the wallet
+  is unable to construct the transaction without selecting funds from the transparent
   pool, so the impact of this change is that for such transactions, the user must specify
   `AllowFullyTransparent`.
 
+Removal of `-mintxfee`
+-----------
```

Comment:
```suggestion
----------------------
```

Reply:
I noticed that the other lines are short, too (which is why I figured we just did a set length) – should I extend all of them?

Reply:
Nevermind – it’s just that I view Markdown in a proportional font, so they didn’t seem to align.